nextflow.config

```diff
@@ -35,9 +55,56 @@ params {
   config_profile_url = false
 }
 
-// Container slug. Stable releases should specify release tag!
-// Developmental code should specify :dev
-process.container = 'nfcore/smartseq:dev'
+
+process {
+  // Container slug. Stable releases should specify release tag!
+  // Developmental code should specify :dev
+//  container = 'nfcore/smartseq:dev'
+  container = 'icbi/smartseq:dev'
+  memory = "16GB"
+  cpus = 1
+
+  // Retry failed processes once 
+  maxRetries = 0
+  errorStrategy = 'retry'
+
+  // Per-process configuration
+  withName:makeSTARindex {
```

Comment:
This section should be in `base.config` where only the resources are defined for individual labels and/or steps where required. The nextflow config just defines the basic container and should follow a bit more the template 👍